src/muya/lib/utils/importMarkdown.js

```diff
@@ -157,9 +157,10 @@ const importRegister = ContentState => {
           const lang = (infostring || '').match(/\S*/)[0]
 
           value = text
-          if (value.endsWith('\n')) {
-            value = value.replace(/\n+$/, '')
-          }
+          // Fix: #1265 and remove codes bellow.
```

Comment:
We could add a option to trim leading and ending new lines in code blocks? Normally this is not a problem to remove the last (unused) new lines.

Reply:
Good idea! The default value is not trim?

Reply:
Well, it's up to you but I think always trim is the way to go.